workers/unity/Assets/Playground/Scripts/Player/ProcessLaunchCommandSystem.cs

```diff
@@ -18,7 +18,8 @@ private struct LaunchCommandData
             [ReadOnly] public EntityArray Entity;
             public ComponentDataArray<SpatialOSLauncher> Launcher;
 
-            [ReadOnly] public ComponentArray<CommandRequests<Generated.Playground.Launcher.LaunchEntity.Request>> CommandRequests;
```

Comment:
Is this PR going to fix this file flip flopping

Reply:
Yes it should by explicitly setting the line-length.